server/src/com/cloud/network/vpn/Site2SiteVpnManagerImpl.java

```diff
@@ -426,72 +426,52 @@ public Site2SiteCustomerGateway updateCustomerGateway(UpdateVpnCustomerGatewayCm
                 }
             }
         }
-        String name = cmd.getName();
         String gatewayIp = cmd.getGatewayIp();
         if (!NetUtils.isValidIp(gatewayIp) && !NetUtils.verifyDomainName(gatewayIp)) {
             throw new InvalidParameterValueException("The customer gateway ip/Domain " + gatewayIp + " is invalid!");
         }
-        if (name == null) {
-            name = "VPN-" + gatewayIp;
-        }
         String guestCidrList = cmd.getGuestCidrList();
         if (!NetUtils.isValidCidrList(guestCidrList)) {
             throw new InvalidParameterValueException("The customer gateway peer cidr list " + guestCidrList + " contains an invalid cidr!");
         }
-        String ipsecPsk = cmd.getIpsecPsk();
         String ikePolicy = cmd.getIkePolicy();
-        String espPolicy = cmd.getEspPolicy();
         if (!NetUtils.isValidS2SVpnPolicy("ike", ikePolicy)) {
-            throw new InvalidParameterValueException("The customer gateway IKE policy" + ikePolicy + " is invalid!  Verify the required Diffie Hellman (DH) group is specified.");
+            throw new InvalidParameterValueException("The customer gateway IKE policy" + ikePolicy + " is invalid! Verify if the required Diffie Hellman (DH) group is specified.");
```

Comment:
can we make this text slightly different then the one above so it will be easier to search when one of them occur? How about adding a reason/context of the use of the invalid policy?